README.md

```diff
@@ -1,5 +1,4 @@
-Ballerina GraphQL Library
-===================
+# Ballerina GraphQL Library
```

Comment:
Will these badges renders on central readme page?

Reply:
No, this is the root-level README file, which is shown in the repository. The other README.md, which resides in the `ballerina` directory will be shown in the central, which does not contain the badges.